docs/docs/components.mdx

```diff
@@ -212,8 +212,13 @@ now implements its behavior.
       # Pre-Trained weights to be loaded
       model_weights: "rasa/LaBSE"
 
-      # An optional path to a specific directory to download and cache the pre-trained model weights.
-      # The `default` cache_dir is the same as https://huggingface.co/transformers/serialization.html#cache-directory .
+      # An optional path to a directory from which
+      # to load pre-trained model weights.
+      # If the requested model is not found in the
+      # directory, it will be downloaded and
+      # cached in this directory for future use.
+      # The default cache_dir is the same as
+      # https://huggingface.co/transformers/serialization.html#cache-directory
```

Comment:
The link:

https://huggingface.co/transformers/serialization.html#cache-directory

needs to be updated. There's no mention of cache_dir in the page linked.

Reply:
Great catch, thank you! I wasn't able to find an appropriate docs page, so named the constant instead. What do you think?

Reply:
What do you think of this doc to describe `cache_dir`?

https://huggingface.co/transformers/installation.html#caching-models

Reply:
That would be the perfect page, but we haven't updated our transformers to that version yet, we are still capped at `<2.12`.. In our version the default path is slightly different (`~/.cache/torch/transformers`)

I found [this](https://huggingface.co/transformers/v2.11.0/serialization.html#cache-directory), but it also isn't quite right, since we want the suffix to be `transformers`. When I do a site-specific search (`".cache/torch/" transformers site:huggingface.co/ 2.11`) nothing relevant comes up..